html/browsers/browsing-the-web/back-forward-cache/README.md

```diff
@@ -0,0 +1,50 @@
+# How to write back-forward cache tests
+
+In the back-forward cache tests, the main test HTML usually
+
+1. Opens new executor Windows using `window.open()` + `noopener` option,
+   because less isolated Windows (e.g. iframes and `window.open()` without
+   `noopener` option) are often not eligible for back-forward cache (e.g.
+   in Chromium).
+2. Injects scripts to / receives values from the executor Windows via
```

Comment:
nit: what means "/", should you remove it?

Reply:
Rephrased.